src/app/desktop/windows/terminal/terminal-states.ts

```diff
@@ -1059,6 +1059,10 @@ export class DefaultTerminalState extends CommandTerminalState {
             if (differentServiceAttacked) {
               startAttack();
             }
+          }, (err) => {
+              if (err.message === 'service_not_running') {
+                this.terminal.outputText('This Computer or Service is not Online.');
```

Comment:
I would probably change the error message to the following:
```suggestion
                this.terminal.outputText('Target service is unreachable.');
```